src/lib.rs

```diff
@@ -203,6 +203,13 @@ pub struct Config {
     /// Maximum number of steps a single iteration of a test can take, and how to react when the
     /// limit is reached
     pub max_steps: MaxSteps,
+
+    /// Time limit for an entire test. If set, calls to [`Runner::run`] will return when the time
+    /// limit is reached or the [`Scheduler`](crate::scheduler::Scheduler) chooses to stop (e.g., by
```

Comment:
Nit: the check below is `start.elapsed() > t` so this should read "limit is exceeded" (or make the check below use `>=` instead of `>`

Reply:
Good catch, fixed (and also fixed a similar nit in the same comment about when the limit is checked).